components/organisms/ProfileHeader.tsx

```diff
@@ -0,0 +1,67 @@
+import { LeftChevronSVG } from "@ensdomains/thorin";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+export enum Tabs {
+  Profile = "profile",
+  Subdomains = "subdomains",
+}
+
+export const ProfileHeader = () => {
+  const router = useRouter();
+  const { name } = router.query; // Dynamic route parameter
+  const { tab } = router.query; // Query parameter
+
+  // To handle tab changes and update the route
+  const handleTabChange = (newTab: string) => {
+    router.push({
+      pathname: `/domains/${name}`,
+      query: { tab: newTab },
+    });
+  };
+
+  useEffect(() => {
+    if (!tab) {
+      handleTabChange("profile");
+    }
```

Comment:
@eduramme can you please apply the feedbacks already given [here](https://github.com/blockful-io/external-resolver-dapp/pull/163) in this PR? THX!